var/spack/repos/builtin/packages/py-morphio/package.py

```diff
@@ -22,6 +22,8 @@ class PyMorphio(PythonPackage):
     version('2.1.2', tag='v2.1.2', submodules=True, get_full_repo=True)
     version('2.0.8', tag='v2.0.8', submodules=True, get_full_repo=True)
 
+    variant('mpi', default=True, description="Use parallel HDF5 when needed")
```

Comment:
no.

Reply:
Thanks, you're right, I was holding the fix to allow the previous jenkins build to finish.